src/Workspaces/Core/Portable/Classification/Classifier.cs

```diff
@@ -92,7 +92,7 @@ public static class Classifier
             foreach (var span in classifiedSpans)
             {
                 // If there is space between this span and the last one, then add a space.
-                if (startPosition != span.TextSpan.Start)
+                if (startPosition < span.TextSpan.Start)
```

Comment:
was this change necessary? Or did you just feel it better conveyed the intent of the comment?

Reply:
Both =)

Since additive spans overlap their previous span then they will trigger this and add additional spaces.